src/seedu/addressbook/AddressBook.java

```diff
@@ -205,18 +205,22 @@
      * referenced by the high-level method below.
      * ====================================================================
      */
-
+    
     public static void main(String[] args) {
         showWelcomeMessage();
         processProgramArgs(args);
         loadDataFromStorage();
-        while (true) {
+        userInputsAndExecutions();
+    }
+
+	public static void userInputsAndExecutions() {
+		while (true) {
```

Comment:
Please fix the indentation and change the tab to spaces in Eclipse, see [here](http://stackoverflow.com/questions/407929/how-do-i-change-eclipse-to-use-spaces-instead-of-tabs)